users/service_test.go

```diff
@@ -1075,7 +1075,7 @@ func TestUpdateClientRole(t *testing.T) {
 			updateRoleResponse:   mgclients.Client{},
 			token:                validToken,
 			deletePolicyErr:      svcerr.ErrMalformedEntity,
-			err:                  svcerr.ErrDeletePolicies,
```

Comment:
@JeffMboya 
Instead of this changing this test response, Please add new test case for invalid / non existance user